### DISCUSSION TITLE
Content Sync - Single page sync (no children pages)

### DISCUSSION THREAD

**Discussion by tim-grimes-graco** [Adobe-Consulting-Services/acs-aem-commons#3417] [Ideas]
Hey! Love the content sync feature but I'm noticing it doesn't let you sync one page (ignoring recursion/children pages). In Package Manager, this can be accomplished by syncing just the /jcr:content node but that doesn't seem to work in here. Might be nice to add a checkbox option to not recurse the tree.

**Comment by YegorKozlov** [comment]
It shouldn't be too hard to support. I'll see what I can do by next release.

**Reply by YegorKozlov** [reply]
https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/3419

**Comment by YegorKozlov** [comment]
Sync-ing a single page is supported since 6.9.0
